### DISCUSSION TITLE
It is possible to use an own HttpClient in the HttpTransferManager?

### DISCUSSION THREAD

**Discussion by AntonS127** [shinyorg/shiny#1108] [Q&A] [answered]
It is possible to use a HttpClient from the HttpClient factory in the HttpTransferManager?

**Comment by aritchie** [comment, marked as answer]
No.  The platform implementations don't currently use httpclient.

**Reply by AntonS127** [reply]
Thank you very much.

I try to make a download request with HttpTransferRequest.Post = HttpMethod.Post on iOS simulator.
I set the string PostData, content length and type but I don't see the data in the request on server side.

Is it possible to send the POST data with the HttpTransferRequest?

**Reply by aritchie** [reply]
PostData is sent raw.  If you need it to be key/value, you have to make the necessary format yourself.  I would encourage you to look through the code to get an answer to questions like these going forward:  https://github.com/shinyorg/shiny/blob/260-dev/src/Shiny.Net.Http/Platforms/iOS/HttpTransferManager.cs
